plugins/modules/email_alert.py

```diff
@@ -244,6 +242,7 @@ def main() -> None:
                 required=True,
             ),
         ),
+        required_if=[("state", "present", ("email_new",))],
```

Comment:
I think this does not work as it should. "Test invalid parameter combination - email_new with state=absent" commit - was this desired result. E.g. fail if email_new is set and state==absent.

Reply:
I agree, it doesn't work as it should. The problem is, that the ``email_new`` param has default value set to ``""``. If that default is removed, than it partially works as it should, but another problem arises - ``email_new`` is not required even when ``state=present``.